packages/next/client/link.tsx

```diff
@@ -180,7 +180,9 @@ class Link extends Component<LinkProps> {
         if (!success) return
         if (scroll) {
           window.scrollTo(0, 0)
-          document.body.focus()
+          // set focus back to the body element
+          const activeElement = document.activeElement as HTMLElement
+          activeElement.blur()
```

Comment:
We should check if this exists right? What if it's an element that doesn't support `blur()`, is that possible?

Reply:
AFAIK, there's always one active element, but of course, it doesn't hurt to add a check 👍

`blur()` is a standard method on all HTMLElements: https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/blur

Reply:
Sure, but you said the type is actually `Element` not `HTMLElement` in a previous comment: https://github.com/zeit/next.js/pull/7693#issuecomment-506532399.